src/redux/books/books.js

```diff
@@ -0,0 +1,22 @@
+// Actions
+const REMOVE = '/src/redux/books/REMOVE';
+const ADD = '/src/redux/books/ADD';
```

Comment:
- [ ] Nice job Please make sure that your actions follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux) according to that you have action types in the form` my-app/reducer/ACTION_TYPE` You have something like `const REMOVE = 'Nyame-Bookstore/books/REMOVE'` and `const ADD = 'Nyame-Bookstore/books/ADD'`. Please fix it :+1: 
- [ ] From the project requirements you should Set an initial state with an empty array then creates and Exports **Action Creators**. (`addBook` and `remove book` functions) , you export well your actions creators but can see you array . Please fix :+1:  find below an example of implementation :arrow_down: 

```
/ ACTIONS
const ADD_BOOK = 'bookstore/books/ADD_BOOK';
const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';

const initialState = [];

// REDUCER
const booksReducer = (state = initialState, action) => {
  switch (action.type) {
    case ADD_BOOK:
      return [...state, action.paylod];
    case REMOVE_BOOK:
      return [...state.filter((book) => book.id !== action.payload.id)];
    default:
      return state;
  }
};

```